.github/workflows/sycl-windows-build.yml

```diff
@@ -104,6 +104,7 @@ jobs:
         echo "C:\Program Files\Git\usr\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
         echo "CCACHE_DIR=D:\github\_work\cache\${{ inputs.build_cache_suffix }}" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
         echo "CCACHE_MAXSIZE=10G" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
+        echo "LIT_OPTS='-j$env:NUMBER_OF_PROCESSORS $LIT_OPTS'" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
```

Comment:
who sets `NUMBER_OF_PROCESSORS`?

Reply:
it's preset by powershell itself i guess

https://www.computerperformance.co.uk/powershell/environmental-variables/

```
 NUMBER_OF_PROCESSORS 4
```